README.md

```diff
@@ -34,10 +34,13 @@ When finished, don't forget to remove the installation disk from your machine. O
 Disable the dhcp server (on the LAN interface) of your pfSense install and you're good to go. Start the ssh-daemon, login and setup public-key authentication (for the root user). Execute `pkg install rsync` and create a script to upload your changes from your development environment to your pfSense install:
 
 ```bash
-#!/bin/sh
+#! /bin/sh
```

Comment:
Okay, but if everyone would commit their personal preference this line will be changed a lot ;)